ui/revenuecatui/build.gradle

```diff
@@ -1,4 +1,6 @@
 plugins {
+    id 'maven-publish'
```

Comment:
This shouldn't be needed either I think, just uncommenting the code below should apply the plugin